activities/Chess.activity/package.json

```diff
@@ -6,5 +6,8 @@
       "sugar-web": "github:sugarlabs/sugar-web/master",
       "domReady": "github:requirejs/domReady/2.0.1"
     }
+  },
+  "dependencies": {
+    "js-ini": "^1.6.0"
```

Comment:
This dependencies is need one time (for migration) but should not stay here.

Reply:
Removed the dependency js.ini from package.json.